shared/util/service-worker/service-worker.js

```diff
@@ -3,16 +3,29 @@
  * while they are not viewing a stream.
  */
 
-import { TRUFFLE_FIREBASE_CONFIG } from "../firebase/config.js";
-import { initializeApp } from "https://npm.tfl.dev/firebase@9.9.4/app";
-import { getMessaging } from "https://npm.tfl.dev/firebase@9.9.4/messaging/sw";
+importScripts(
+  "https://www.gstatic.com/firebasejs/9.2.0/firebase-app-compat.js"
+);
+importScripts(
+  "https://www.gstatic.com/firebasejs/9.2.0/firebase-messaging-compat.js"
+);
 
-initializeApp(TRUFFLE_FIREBASE_CONFIG);
+firebase.initializeApp({
```

Comment:
is this intentional?

does `import { TRUFFLE_FIREBASE_CONFIG } from "../firebase/config.ts";` work?